setup/tue-functions.bash

```diff
@@ -202,27 +202,24 @@ function _tue-repo-status
     else
         # Try git
 
-        cd $pkg_dir
-        res=$(git status . --short --branch 2>&1)
+        res=$(git -C $pkg_dir status . --short --branch 2>&1)
         if [ $? -eq 0 ]
         then
             # Is git
             if echo "$res" | grep -q -E 'behind|ahead' # Check if behind or ahead of branch
             then
                 status=$res
             else
-                status=`git status . --short`
-            fi
-
-            local current_branch=`git rev-parse --abbrev-ref HEAD`
-            if [ $current_branch != "master" ]  && [ $current_branch != "develop" ] && [ $current_branch != "indigo-devel" ] && [ $current_branch != "kinetic-devel" ] && [ $current_branch != "toolchain-2.9" ] && ! _robocup_branch_allowed $current_branch
```

Comment:
What happend to "_robocup_branch_allowed $current_branch"??

Reply:
Eeehm, whups.